src/components/balance/ModalTransferAmount.vue

```diff
@@ -90,16 +95,22 @@
                     off:hidden
                   -->
                   <div
+                    v-if="openOption"
                     class="block absolute mt-1 w-full rounded-md bg-white dark:bg-darkGray-800 shadow-lg z-10 border border-gray-200 dark:border-darkGray-600"
                   >
                     <ul
                       class="max-h-56 rounded-md py-1 text-base overflow-auto focus:outline-none"
                     >
                       <!-- Select option -->
-                      <ModalSelectAccountOption />
-                      <!-- <ModalSelectAccountOption />
-                      <ModalSelectAccountOption />
-                      <ModalSelectAccountOption /> -->
+                      <ModalSelectAccountOption
+                        v-for="(account, index) in allAccounts"
```

Comment:
In the transfer target address option, the user should be able to input their own address that is not saved as well.

Reply:
does that mean that we need separate input textfield for address out of dropdown field? @hoonsubin

Reply:
We can do that, or we can allow the user to directly provide their own input within the dropdown field. Something like this in the polkadot-js apps portal
![Screenshot from 2021-04-27 14-12-12](https://user-images.githubusercontent.com/40356749/116188571-9ac27180-a762-11eb-84e7-23ce1e5f39ec.png)